src/Education/RuleHelpToolWindow.cs

```diff
@@ -23,9 +23,10 @@
 using System.Windows.Documents;
 using Microsoft.VisualStudio.Shell;
 using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Education.Controls;
 using SonarLint.VisualStudio.Integration;
 
-namespace SonarLint.VisualStudio.Education.Commands
+namespace SonarLint.VisualStudio.Education
```

Comment:
I also moved the tool window out of the `Commands` folder. I didn't feel like it really belonged in either the `UI` folder or the `Controls` folder so I moved it to the top level. That way at least it is more visible.